src/components/sortable/helpers.ts

```diff
@@ -16,3 +16,23 @@ export const getPreviewStyles = (clientOffset: XYCoord | null): CSSProperties =>
     WebkitTransform: transform,
   };
 };
+
+export const calculateCursorBasedDropIndex = (
+  fromIndex: number,
+  targetIndex: number,
+  isTopZone: boolean,
+): number => {
```

Comment:
- Let's create object when number of params > 2
- No need to specify retrun type - TypeScript can infer

```suggestion
export const calculateCursorBasedDropIndex = ({
  fromIndex,
  targetIndex,
  isTopZone,
}: {
  fromIndex: number,
  targetIndex: number,
  isTopZone: boolean,
}) => {
```